### PR TITLE
Fix E2E test

### PR DESCRIPTION
**Pull request by gigitux**
Currently, various e2e tests are failing. This PR aims to fix these failing tests.

### Changes

- Format various fixtures.
- Change logic for retrieving the cart and checkout page IDs.
- Adjust logic of mini cart translation tests.

#### Automated Tests
* [ ] Changes in this PR are covered by Automated Tests.
  * [ ] Unit tests
  * [x] E2E tests

Look up the checks via https://github.com/woocommerce/woocommerce-blocks/pull/7206/checks or at the bottom of this page and verify that all tests are passing.

### CODE REVIEW

File: tests/e2e/specs/shopper/cart-checkout/checkout.test.js [hunk 1]
```diff
@@ -44,16 +44,6 @@ describe( 'Shopper → Checkout', () => {
 		await shopper.block.emptyCart();
 	} );
 
-	it( 'User can view empty cart message', async () => {
```
Comment (gigitux) [inline tests/e2e/specs/shopper/cart-checkout/checkout.test.js:47]: It seems that this test isn't necessary anymore (p1665157378161879-slack-C8X6Q7XQU)

File: tests/e2e/fixtures/fixture-loaders.js [hunk 2]
```diff
@@ -49,7 +49,7 @@ const setupSettings = ( fixture = fixtures.Settings() ) =>
 	} );
 
 const setupPageSettings = () => {
-	axios.get( WPAPI ).then( ( response ) => {
+	axios.get( `${ WPAPI }?per_page=100` ).then( ( response ) => {
```
Comment (gigitux) [inline tests/e2e/fixtures/fixture-loaders.js:52]: This is necessary for get all the pages.

File: tests/e2e/specs/shopper/mini-cart.test.js [hunk 3]
```diff
@@ -552,7 +548,9 @@ describe( 'Shopper → Mini Cart', () => {
 				( el ) => el.href
 			);
 
-			expect( checkoutUrl ).toMatch( SHOP_CHECKOUT_PAGE );
+			expect( checkoutUrl ).toMatch(
+				'http://localhost:8889/checkout-block/'
```
Comment (alexflorisca) [inline tests/e2e/specs/shopper/mini-cart.test.js]: Would be great if we can avoid hardcoding server and port numbers here. How comes `SHOP_CHECKOUT_PAGE` doesn't work here?

Comment (alexflorisca) [reply tests/e2e/specs/shopper/mini-cart.test.js]: Maybe you can use the `SHOP_CHECKOUT_BLOCK_PAGE` defined [here](https://github.com/woocommerce/woocommerce-blocks/pull/7206/files#diff-5c358b5fd8e2d46aa29a0dfd50e7059885e64c192a88cf560245f4f20540b07dR43)

**Review by albarin — APPROVED** [review]

**Review by alexflorisca — CHANGES_REQUESTED** [review]

**Review by alexflorisca — APPROVED** [review]
Looks good. The empty checkout test might still be valid, but as mentioned on slack p1665396608032039/1665389378.477259-slack-C02UBB1EPEF, I'll investigate that in a different PR

**Pull request merged**